### DISCUSSION TITLE
Far-Field intensity

### DISCUSSION THREAD

**Discussion by ghost** [NanoComp/meep#2281] [Q&A]
Dear MEEP developers,
First of all, thank you for keeping this wonderful project . I am a MEEP beginner. I would like to use MEEP for simulating far-field intensivty like figure 2 (f)-(g) -(h) in this paper (DOI: 10.1088/2040-8978/17/11/114011#top). Is it posible? Can you instruct me? Thank a lot.

**Comment by smartalecH** [comment]
Sure, check out the 5 different examples we have on Near-to-far transformations:

https://meep.readthedocs.io/en/latest/Python_Tutorials/Near_to_Far_Field_Spectra/
